src/datasets/bigquery/clients_last_seen/intro.md

```diff
@@ -0,0 +1,55 @@
+The `clients_last_seen` dataset is useful for efficiently determining exact
+user counts such as [DAU and MAU](../../../cookbooks/dau.md).
+
+It does *not* use approximates, unlike the HyperLogLog algorithm used in the
+[`client_count_daily` dataset](/datasets/batch_view/client_count/reference.md),
+and it includes the most recent values in a 28 day window for all columns in
+the [`clients_daily` dataset](/datasets/batch_view/clients_daily/reference.md).
+
+This dataset should be used instead of `client_count_daily` in BigQuery,
+and is not currently available outside of BigQuery.
+
+#### Content
+
+For each `submission_date` this dataset contains one row per `client_id`
+that appeared in `clients_daily` in a 28 day window including
+`submission_date` and preceding days.
+
+The `days_since_seen` column indicates the difference between `submission_date`
+and the most recent `submission_date` in `clients_daily` where the `client_id`
+appeared.
+
+Other `days_since_` columns use the most recent date in `clients_daily` where
+a certain condition was met. If the condition was not met for a `client_id` in
+a 28 day window `NULL` is used. For example `days_since_visited_5_uri` uses the
+condition `scalar_parent_browser_engagement_total_uri_count_sum >= 5`. These
+columns can be used for user counts where a condition must be met on any day
+in a window instead of using the most recent values for each `client_id`.
+
+The rest of the columns use the most recent value in `clients_daily` where
+the `client_id` appeared.
+
+#### Background and Caveats
+
+User counts generated using `days_since_seen` only reflect the most recent
+values from `clients_daily` for each `client_id` in a 28 day window. This means
+[Active MAU](../../../cookbooks/active_dau.md)
+as defined cannot be efficiently calculated using `days_since_seen` because if
+a given `client_id` appeared every day in February and only on February 1st had
+`scalar_parent_browser_engagement_total_uri_count_sum >= 5` then it would only
+be counted on the 1st, and not the 2nd-28th. Active MAU can be efficiently and
+correctly calculated using `days_since_visited_5_uri`.
+
+MAU can be calculated over a `GROUP BY submission_date[, ...]` clause using
+`COUNT(*)`, because there is exactly one row in the dataset for each
```

Comment:
Perhaps it's worth pointing out that these three are equivalent:

```
count(days_since_seen < 28)
count(days_since_seen)
count(*)
```

for the specific case of calculating general MAU, but the most specific form is necessary to express both a different window and a different usage criterion.